Core/Core/Planner/Model/UseCase/GetPlannables.swift

```diff
@@ -88,11 +89,42 @@ extension APICalendarEvent: PlannableItem {
     public var isHidden: Bool { hidden == true }
 }
 
+extension APIPlannerNote: PlannableItem {
+    public var plannableID: String { id }
+    public var plannableType: PlannableType { .planner_note }
+    public var htmlURL: URL? { nil }
+
+    public var context: Context? {
+        if let courseID = course_id {
+            return Context(.course, id: courseID)
+        }
+        if let userId = user_id {
+            return Context(.user, id: userId)
+        }
+        return nil
+    }
+
+    public var contextName: String? { nil }
+    public var plannableTitle: String? { title }
+    public var date: Date? { todo_date }
+    public var pointsPossible: Double? { nil }
+    public var isHidden: Bool { false }
+}
+
 public class GetPlannables: UseCase {
     public typealias Model = Plannable
     public struct Response: Codable, Equatable {
+        static let empty = Response(plannables: [], calendarEvents: [], plannerNotes: [])
+        static func make(
```

Comment:
I think this method could be removed if the properties below would be `var`-s.
Plus `make()` is (almost?) always an debug-build-only method in the project.